PsevdoTwwiterApp/src/main/webapp/resources/styles.css

```diff
@@ -0,0 +1,500 @@
+body{
+width: 100%;
+margin: 0;
+font-size: 18px;
+position: relative;
+
+}
+.Log {
+float: right;
+margin-left: 1350px;
+border-radius: 25%;
+width: 70px;
+height: 30px;
+}
+
+h1 {
+margin-top: 5px;
+font-family: 'Roboto';
+font-size:34px;
+width: 250px;
+padding-top: 15px;
+padding-bottom: 70px;
+height: 35px;
+padding-left: 60px;
+margin-bottom: 20px;
+position: fixed;
+padding-bottom:20px;
+float:left;
+}
+
+form {
+list-style-type: none;
+margin-top: 20px;
+height: 80%;
+padding-left: 0;
+margin-left: 0;
+padding-top: 50px;
+width: 25%;
+position: fixed;
+display: block;
+font-family: 'Roboto';
+float: left;
+font-size: 30;
+overflow: none;
```

Comment:
`overflow` can't be `none`
`overflow: auto | hidden | scroll | visible | inherit`